def division(a, b):
    x=a; y=b
    try:
        return x / y
    except ZeroDivisionError:
        print("Zero division error!")
        




print(division(189, 0))
# 5.25

"""Напишите функцию division(a, b), которая принимает на вход два числа, делимое (a) и делитель (b), и возвращает частное.

Необходимо реализовать внутри функции отлов исключения ZeroDivisionError на случай, если пользователь при вызове функции попытается поделить на ноль.

Если в процессе выполнения функции было поймано исключение ZeroDivisionError, то функция должна вывести на экран сообщение "Zero division error!" с помощью функции print(), а затем вернуть значение None из функции.

Примеры работы программы:


print(division(189, 36))
# 5.25
print(division(1, 0))
# Zero division error!
# None"""